modules/client/src/main/java/org/apache/ignite/internal/client/compute/ClientCompute.java

```diff
@@ -48,6 +48,8 @@
  * Client compute implementation.
  */
 public class ClientCompute implements IgniteCompute {
+    private static final String DEFAULT_SCHEMA_NAME = "PUBLIC";
```

Comment:
we have a few places with default schema names. May be we could use a single place?

Reply:
Can you advise such place please?

Reply:
right now it could be IgniteNameUtils, later it possible to migrate to configuration.